### PR TITLE
Create LinearRegression_beyzaNur.ipynb

### PR DESCRIPTION
**Pull request by beyzanurdeniz**
@bkavlak @isikz

### CODE REVIEW

File: task0/LinearRegression_beyzaNur.ipynb [hunk 1]
```diff
@@ -0,0 +1,1586 @@
+{
```
Comment (bkavlak) [inline task0/LinearRegression_beyzaNur.ipynb:1]: <p>Are you sure that you included the right file? I cant see the solutiıns here. It seems that you adjusted the code above, but did you reported it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='29'/>

Comment (beyzanurdeniz) [reply task0/LinearRegression_beyzaNur.ipynb:1]: I updated the file with the answers, I think I had a problem with saving first time maybe.

Comment (bkavlak) [reply task0/LinearRegression_beyzaNur.ipynb:1]: If an issue is solved here, we click "Resolve conversation" to make this PR pretty. Feel free to do that.

File: task0/LinearRegression_beyzaNur.ipynb [hunk 2]
```diff
@@ -0,0 +1,1600 @@
...
```
Comment (bkavlak) [inline task0/LinearRegression_beyzaNur.ipynb:1]: <p>ANSWERS:</p><p>1 - The score are as the following:</p><p>A. ToNthHouse=10 -&gt; 0.752</p><p>B. ToNthHouse=14 -&gt; 0.116</p><p>2 - Outliers are:</p><p>A. data(4)</p><p>B. data(4, 10, 11, 12, 13)</p><p>3 - For situation A, linear regression fits to data set; but in situation B, we might need higher degree polynomial approach.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='30'/>

Comment (bkavlak) [inline task0/LinearRegression_beyzaNur.ipynb:1]: <p>You may print results in a more human readable format:</p><pre class="ql-syntax" spellcheck="false">score = model.score(ageTransformed,Price.reshape(-1,1))
print(f"The regression score is ~ {score:.4f}")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='41'/>

**Pull request closed without merging**